members/apoorva-teja.css

```diff
@@ -0,0 +1,90 @@
+@import "compass/css3";
+body,
```

Comment:
Can you try sectioning out the css tags using SMACSS? So basic html tags would go under 'base', layout tags would go under 'layout', really specific tags go under 'module', etc.

Reply:
Honestly I haven't worked on CSS. It's copy paste from flex-box site.  Can exclude from commit.